.github/.wordlist.txt

```diff
@@ -1438,6 +1441,7 @@ TrustedRootCertificates
 tsan
 TSG
 tsv
+tt
```

Comment:
this is not a word ... why did we need to add it? was it a typo for `to`? I don't see it anymore.

Reply:
I believe it pick up the 'tt' from  'csg-tt-test-plans'.  For example, 'csg' was already in the word list.

Reply:
could we add backticks on that?

Reply:
Yes, I believe we ignore any URLs or backticks: https://github.com/project-chip/connectedhomeip/blob/master/.spellcheck.yml#L36